src/include/add-on/add-on-workflow.rb

```diff
@@ -1267,23 +1271,21 @@ def AdjustRepositoryInfo(info)
     def GetRepoInfo(this_product, all_products)
       ret = { "IDs" => [], "URLs" => [], "aliases" => [] }
 
-      product_arch = Ops.get_string(this_product.value, "arch", "")
-      product_name = Ops.get_string(this_product.value, "name", "")
-      product_version = Ops.get_string(this_product.value, "version", "")
+      product_arch = this_product.value.arch || ""
```

Comment:
I think we can remove that default values (`|| ""`), these were forced by the old YCP calls which required to specify the default. Pkg bindings should always return a valid String there.